### PR TITLE
More mappings

### PR DESCRIPTION
**Pull request by pyrox0**
Updates a bunch of stuff that got unmapped from 24w12a and 24w13a, plus datafixer, resource pack, structure, and network mappings.

### CODE REVIEW

File: mappings/net/minecraft/advancement/criterion/RecipeCraftedCriterionTrigger.mapping [hunk 1]
```diff
@@ -17,3 +17,5 @@ CLASS net/minecraft/unmapped/C_thfyxhzz net/minecraft/advancement/criterion/Reci
 		METHOD m_srqmbfmi matches (Lnet/minecraft/unmapped/C_ncpywfca;Ljava/util/List;)Z
 			ARG 1 recipeId
 			ARG 2 stacks
+		METHOD m_zrhvqutd craftedCraftedItem (Lnet/minecraft/unmapped/C_ncpywfca;)Lnet/minecraft/unmapped/C_rzypsigz;
```
Comment (supersaiyansubtlety) [inline mappings/net/minecraft/advancement/criterion/RecipeCraftedCriterionTrigger.mapping]:
```suggestion
		METHOD m_zrhvqutd crafterCraftedItem (Lnet/minecraft/unmapped/C_ncpywfca;)Lnet/minecraft/unmapped/C_rzypsigz;
```

**Review by supersaiyansubtlety — CHANGES_REQUESTED** [review]
looks good, just 1 typo

File: mappings/net/minecraft/advancement/criterion/RecipeCraftedCriterionTrigger.mapping (hunk 1 above)
Comment (pyrox0) [reply mappings/net/minecraft/advancement/criterion/RecipeCraftedCriterionTrigger.mapping]: Fixed!

**Review by supersaiyansubtlety — APPROVED** [review]

**Review by eristhea — APPROVED** [review]
lovely!

**Pull request merged**